core/templates/dev/head/css/oppia.css

```diff
@@ -65,6 +65,10 @@ button.md-button.md-default-theme.oppia-learner-continue-button:hover {
   background-color: #115FD4;
 }
 
+/* Attributes for material-icons. */
+.material-icons{
```

Comment:
@seanlip I checked other icons and are not affected by this class
It only aligns the Play icon in the middle as expected 
Thanks!